### PR TITLE
Crystal v1 bump

### PR DESCRIPTION
**Pull request by GabFitzgerald**

### CODE REVIEW

File: shard.yml [hunk 1]
```diff
@@ -20,6 +20,6 @@ development_dependencies:
   webmock:
     github: manastech/webmock.cr
 
-crystal: 0.36.1
+crystal: ">=0.36.1"
```
Comment (dukenguyenxyz) [inline shard.yml]:
```suggestion
crystal: "~> 1.0"
```

**Comment by dukenguyenxyz** [comment]
can you manually rerun ci build on travis for this branch please. the ci is not running for the branch

File: shard.yml (hunk 1 above)
Comment (GabFitzgerald) [reply shard.yml]: unless you want it to be backward compatible?

Comment (dukenguyenxyz) [reply shard.yml]: we have bumped all of our libs to be "~> 1.0"

**Review by dukenguyenxyz — APPROVED** [review]

**Pull request merged**